src/controllers/routers/cases.js

```diff
@@ -74,14 +91,245 @@ function sanitizedCase(json) {
   return cases;
 }
 
+/**
+ * Sanitize data from query params.
+ * Call before an insert or an update.
+ * @param {Object} json - Unsanitised query params
+ * @return {Object} query - Sanitised query params
+ */
+function sanitizedQuery(json) {
```

Comment:
since with_paging is checked against "true" / "false" all throughout the code; we need to convert the with_paging value sent from client to lower case.

Reply:
added in sanitizedQuery to include toLowerCase() for with_paging